user_list.php

```diff
@@ -29,64 +29,46 @@
 			<!-- import the webpage's javascript -->
 			<!-- nav javascript -->
 			<script src="js/nav.js" defer></script>
-			<!-- back to top button javascript -->
-			<script src="js/backtotop.js" defer></script>
 		</head>
 	
         <body>
-              <!-- logo at top of page-->
-			<center><a href="index.php"><img style="width: 100%; max-width: 100px;" src="images/placeholder.png" alt="TuneIn logo"></a></center>	
-
 			<!-- navigation menu (burger menu)-->
 			<?php
 			  require("nav.php");
 			?>
 			
-                                <div class="aside"><!-- holds the admin menu with links to the different pages only the admin can access -->
```

Comment:
#trialled having an aside which would store all the links to the other admin pages, however, this became redundant as I used a dropdown menu in my second layout concept